about/policy/sgid/index.md

```diff
@@ -10,16 +10,33 @@ categories: []
 date: 2019-12-04 10:01:27 -0500
 ---
 
-The following applies to data in the internal SGID database.
+The following policies apply to data in the internal SGID Enterprise Geodatabase.
```

Comment:
this might be a good spot to explain (maybe 2 sentences) what the sgid is.  i'll see if i can come up with something that we can hash out. 
the definition of the sgid evolves as technology evolves.  in my mind, the current sgid could be composed of (1.) what's in the Open Data SGID (the curated stuff that you describe on this page) and (2.) what's in the sgid index - which might be more like a state of utah gis data catalog with both items in the Open SGID as well as pointers to items that might be of interest, but do not meet our guidelines laid out here on this page.

Reply:
I just pushed a commit with some verbiage. 

I think the full explanation of the differences between the internal SGID, the OpenSGID, ArcGIS Online, and Open Data would be a great candidate for the /gis-data-sharing page. I think we should keep this page focused on the internal SGID and only mention the others when describing how/when data is copied to them from the internal SGID.

Reply:
_[here's a fist shot at it.  it needs some refining.  see what you can do]_

The SGID is composed of both curated and non-curated data. The policies outlined on this page focus on curated data.

_(maybe additional info like this can be displayed below the above statement, but less prominent - italic or a side note)_ 
The entire SGID can be explored using AGRC's SGID search index. Additionally, curated data is found on AGRC's data pages, and is hosted on AGRC's Open Data site and/or Open SGID database.

Reply:
i totally agree about the data page being the place for the full explanation.

Reply:
See #1508 for changes to the data page that explain the different facets of the SGID.